xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/macros.vm

```diff
@@ -63,7 +63,22 @@
  * @param class The class to use.
  *#
 #macro(submitButton $name $shortcut $value $class)
-<input class="btn $!class" type="submit" name="$name"#if($keyboardShortcutsEnabled) title="$shortcut"#end value="$value"/>
+<input class="btn $!class" type="submit" name="$name"#if($keyboardShortcutsEnabled) title="$escapetool.xml($shortcut)"#end value="$value"/>
+#end
+#**
+ * Returns the escaped value of the shortcut. This value can be set in two ways: via user preferences and wiki level translation
```

Comment:
There seems to be no more escaping (which is okay) but the comment needs to be updated.

Reply:
Addressed in https://github.com/xwiki/xwiki-platform/pull/2981/commits/90f8096a8de8cc1f5f547026a343e05486503e3c :+1: